File: Vladrega.Auctions.Domain/AuctionStatus.cs
namespace Vladrega.Auctions.Domain;

/// <summary>
/// Статус акциона
/// </summary>
public enum AuctionStatus
{
    /// <summary>
    /// Неизвестный статус
    /// </summary>
    Unknown = 0,
    
    /// <summary>
    /// Этап создания аукциона
    /// </summary>
    Creation = 1,
    
    /// <summary>
    /// Этап ожидания торгов
    /// </summary>
    WaitBidding = 2,
    
    /// <summary>
    /// Этап торгов
    /// </summary>
    Bidding = 3,
    
    /// <summary>
    /// Аукцион завершен
    /// </summary>
    Complete = 4
}

File: frontend/my-app/src/models/AuctionStatus.ts
export enum AuctionStatus {
    /** Этап создания ауцкциона */
    Creation = 1,
    
    /** Этап ожидания торгов */
    WaitBidding = 2,
    
    /** Идут торги */
    Bidding = 3,
    
    /** Аукцион завершен */
    Complete = 4,
    
    /** Аукцион отменен */
    Canceled = 5
}